tox.ini

```diff
@@ -8,7 +8,8 @@ envlist = py27, py3
 
 [testenv]
 commands =
-    python -m unittest discover -s tests
+    py27: python -m unittest discover -s tests -p 'test_sync*.py'
+    py3: python -m unittest discover -s tests -p 'test*.py'
```

Comment:
The `py3` test would fail for a system that has python version < 3.6 (and maybe 3.7?). Should `py3` in the envlist be changed to `py36` as it would be clearer what is being tested and why it fails? You did something similar elsewhere and maybe the same should be done here. The testenv command could include `py35` as well in case a user wants to test that using the tox `-e` option.